eo-runtime/src/main/java/org/eolang/AtComposite.java

```diff
@@ -71,6 +71,9 @@ public Attr copy(final Phi self) {
     public Phi get() {
         try {
             return this.expr.get(this.rho);
+        } catch (final InterruptedException ex) {
+            System.out.println("Interrupted Exception");
```

Comment:
@levBagryansky Try to not to use the debugging mechanism to output to the console in production code. We can pass this information through the constructor instead

Reply:
@Graur my bad, thank you